packages/amplify-cli-core/src/exeInfo.ts

```diff
@@ -1,7 +1,10 @@
-import { $TSAny } from './types';
+import { $TSAny, $TSObject, LocalEnvInfo } from './types';
+
+// base type
+export type ExeInfo = EnvironmentInfo & InputParameters & PinpointInfo & ProjectInfo;
```

Comment:
This is the line that is causing `ExeInfo.ExeInfo`. Because you are namespacing the types in the `ExeInfo` file as `ExeInfo` in `amplify-cli-core`'s `index.ts`, this is going to require a consumer to write code like this:

```typescript
import { ExeInfo } from 'amplify-cli-core';

let exeInfo: ExeInfo.ExeInfo = {};

```

That's not necessarily awful, but it does feel a bit repetitive. A clearer name, or possibly exporting it as default could be a path forward here.

Reply:
Yeah, this is something I meant to delete. I was speculating that it might be easier to have a type that encompasses the joined classes, but didn't end up using it anywhere. I think I should just delete it?